tailwind.config.js

```diff
@@ -1,7 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
-  darkMode: "dark",
+  darkMode: "class",
```

Comment:
is this needed ?

Reply:
Dark mode wasn’t working without this addition. I got it from the docs of course.

Reply:
I will check if it works without it, and try to understand in general what this property do

Reply:
Hey, i looked up the docs a couple times, I looked everywhere i didnt find a darkMode: "dark" . There is several darkMode variants but dark isnt one of them. I will resolve, if u find anything else please open it up again

Reply:
Sure, if you face problems just leave it, we're planning a separate PR for all the theming issues.